release/util.bzl

```diff
@@ -143,6 +143,8 @@ protos_zip = rule(
             default = [
                 Label("//daml-lf/archive:daml_lf_{}_archive_proto_tar.tar.gz".format(version))
                 for version in LF_VERSIONS
+                # we skip LF 1.12 as it is exactly as LF 1.11 for archive point of view
```

Comment:
I’m not sure that’s a good idea in released artifacts. I think this is perfectly sensible to skip such versions in tests but we should have a released artifact for that version, otherwise it just becomes confusing.